domain/consensus/utils/pow/pow.go

```diff
@@ -78,7 +102,7 @@ func BlockLevel(header externalapi.BlockHeader) int {
 		return constants.MaxBlockLevel
 	}
 
-	proofOfWorkValue := CalculateProofOfWorkValue(header.ToMutable())
+	proofOfWorkValue := NewMinerState(header.ToMutable()).CalculateProofOfWorkValue()
```

Comment:
nit: Use a wrapper

Reply:
That's inside of the `pow` package, do you really feel it requires a wrapper?